bika/lims/browser/__init__.py

```diff
@@ -94,6 +94,8 @@ def ulocalized_time(time, long_format=None, time_only=None, context=None,
                 time_str = ''
 
         return time_str
+    if time is None:
+        return ''
```

Comment:
This piece of code is unnecessary. Note that either if time is none or time is not an instance of DateTime, the function always return an empty string. See line L102 below